src/write_mongodb.c

```diff
@@ -162,7 +171,7 @@ static int wm_initialize(wm_node_t *node) /* {{{ */
     uri_length = strlen(format_string) + strlen(node->user) +
                  strlen(node->passwd) + strlen(node->host) + 5 +
                  strlen(node->db) + 1;
-    if ((uri = malloc(sizeof(char) * uri_length)) == NULL) {
+    if ((uri = calloc(uri_length, sizeof(char))) == NULL) {
```

Comment:
I can change it to 1 (I wanted to use `sizeof(char)` just to be explicit), but on my man page, the number of members/elements is the first argument, and the second argument is the size of each element.